tests/test_object_model/test_tipping_points.py

```diff
@@ -0,0 +1,92 @@
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import pytest
+import tomli
+from flood_adapt.api.static import read_database
+from flood_adapt.config import set_system_folder
+
+from flood_adapt.object_model.tipping_point import TippingPoint
+from flood_adapt.object_model.interface.tipping_points import ITipPoint
+
+
+@pytest.fixture
+def setup_database():
+    # Mock the database setup or configure a test database
+    read_database(
+        rf"C:\\Users\\morenodu\\OneDrive - Stichting Deltares\\Documents\\GitHub\\FloodAdapt-Database",
+        "charleston_full",
+    )
+    set_system_folder(
+        rf"C:\\Users\\morenodu\\OneDrive - Stichting Deltares\\Documents\\GitHub\\FloodAdapt-Database\\system"
+    )
+
+
+def test_tipping_point_creation(setup_database):
+    # Setup
+    tp_dict = {
+        "name": "tipping_point_test",
+        "description": "",
+        "event_set": "extreme12ft",
+        "strategy": "no_measures",
+        "projection": "current",
+        "sealevelrise": [0.5, 1.0, 1.5],
+        "tipping_point_metric": [
+            ("FloodedAll", 34195.0, "greater"),
+            ("FullyFloodedRoads", 2000, "greater"),
+        ],
+    }
```

Comment:
You can put the dict in a fixture and then use it in more than 1 test:

```
@pytest.fixture()
def tp_dict():
      tp_dict = {
        "name": "tipping_point_test",
        "description": "",
        "event_set": "extreme12ft",
        "strategy": "no_measures",
        "projection": "current",
        "sealevelrise": [0.5, 1.0, 1.5],
        "tipping_point_metric": [
            ("FloodedAll", 34195.0, "greater"),
            ("FullyFloodedRoads", 2000, "greater"),
        ],
    }
  return tp_dict
  
test_xx_xx(test_db, tp_dict):
  test_point = TippingPoint.load_dict(tp_dict)
  # the rest of the test
  
```

Reply:
Thank you! Done!